clientconn.go

```diff
@@ -245,6 +245,7 @@ func DialContext(ctx context.Context, target string, opts ...DialOption) (conn *
 
 	// Determine the resolver to use.
 	cc.parsedTarget = grpcutil.ParseTarget(cc.target)
+	unixScheme := strings.HasPrefix(cc.parsedTarget.Endpoint, "unix:")
```

Comment:
Check `cc.target` directly?

In the followup change to set dialer, we may want to do something similar to https://github.com/grpc/grpc-go/blob/master/rpc_util.go#L821:6

Reply:
Using cc.target directly.